reference/providers/kubernetes-v2/index.md

```diff
@@ -7,106 +7,83 @@ sidebar:
 
-## Accomodating Level-based Deployments
+## Accomodating Level-Based Deployments
 
 See the [Kubernetes API
 conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#spec-and-status)
-for a description of an edge- vs. level-based API.
-
-Prior providers in Spinnaker focused on tracking an operation that modified a
-cloud resource. For example, if a resize operation was run, Spinnaker
-would monitor that operation until the specific resize target was met. This
-runs counter to the Kubernetes level-based API, which only cares about the
-desired state of a resource being satisfied. In order to address this, we
-introduced the concept of __Manifest Stability__:
-
-A deployed manifest is consided __stable__ when the Kubernetes
-Controller-Manager no longer needs to modify it, and it is "ready". This
-definition is clearly very dependent on the manifest's kind (e.g. `Service` vs.
-`StatefulSet`), but for example, a `Deployment` is stable when its managed pods
-are updated, available, and ready (meaning they are running your desired
-container, and are serving traffic). Each kind's stability requirements are
-detailed in the [Resource Mapping](#resource-mapping) section.
-
-This concept of Manifest Stability is how Spinnaker ensures that operations
-have succeeded. Since there are a number of reasons why a manifest never
+for a description of edge-based vs. level-based APIs.
+
+Other providers in Spinnaker track operations that modify cloud resources. For example, if you run a resize operation, Spinnaker monitors that operation until the specified resize target is met. This is different from how the Kubernetes level-based API does  it: Kubernetes only tries to satisfy the desired state.
+
+To take advantage of this, the Kubernetes provider v2 assesses "manifest stability." A deployed manifest is considered stable when the Kubernetes controller-manager no longer needs to modify it, and it’s deemed “ready.” This assessment is different, obviously, for different `kind`s of manifests: a `deployment` is stable when its managed pods are updated, available, and ready (running your desired container and serving traffic). A `service` is stable once it is created, unless it is of type `LoadBalancer`, in which case it is considered stable once the underlying load balancer has been created and bound to the `Service`.
+
+This manifest stability is how Spinnaker ensures that operations
+have succeeded. Because there are a number of reasons why a manifest never
 becomes stable (lack of CPU quota, failing readiness checks, no IP for a
-service to bind...) every stage that modifies or deploys a manifest will wait
-until your affected manifests are stable, or timeout after a configurable
+service to bind...) every stage that modifies or deploys a manifest waits
+until your affected manifests are stable, or it times out after a configurable
```

Comment:
Oh I forgot to mention this, could you add that the default is a 30 minute window before timing out?